template/next-ts-chakra-ui/pages/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Head from 'next/head';
+import type { NextPage } from 'next';
+
+import { Home } from '@/components/pages';
+import generateMetaTags from '@/utils/meta';
+
+const Index: NextPage = () => {
```

Comment:
If we are going to use this as our Best Practice Broilerplate, it would be good to probably have a getServerSideProps() typed function in here.

Reply:
Good point, this slipped my mind. Thanks 👍 
I guess I'll add a clock that shows the current time or something 😛

Reply:
Done 👍